scene/main/window.cpp

```diff
@@ -1631,35 +1630,6 @@ bool Window::_can_consume_input_events() const {
 
 void Window::_window_input(const Ref<InputEvent> &p_ev) {
 	ERR_MAIN_THREAD_GUARD;
-	if (EngineDebugger::is_active()) {
```

Comment:
The reason why this code was in Window is to allow stopping execution from any Window. With your current implementation, you can't stop if there is e.g. AcceptDialog and windows are not embedded. Though *probably* it's fine like this.